src/integrationTest/java/uk/gov/hmcts/reform/iacaseapi/Service2ServiceEnabledIntegrationTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.iacaseapi;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource(properties = {"S2S_URL=http://127.0.0.1:8990"})
+public abstract class Service2ServiceEnabledIntegrationTest extends SpringBootIntegrationTest {
```

Comment:
Did you mean to include this here?  Probably from the Asylum-case-refactor branch to be honest as this is rebased on top.

Reply:
So forgot to mention, original I wanted to introduce an integration test using the new framework, but there was something odd going on with an endpoint /lease that meant it was failing on jenkins but passing locally.  I have a suspicion that I might have left the CCD container running when testing locally, so it was using that.  I didnt have time work on them further so left the framework in place as its only in test.  Will look into this as part of the next story and evolve further.

Reply:
as discussed